### PR TITLE
Updating optics integration test requirements

### PR DESCRIPTION
**Pull request by ahsaanyousaf**

### CODE REVIEW

File: feature/platform/tests/optics_power_and_bias_current_test/README.md [hunk 1]
```diff
@@ -1,49 +1,102 @@
...
+# gNMI-1.13: Optics Telemetry, Instant, threshold, and miscellaneous static info
 
 ## Summary
 
-Validate optics input power, output power and bias current.
+Validate optics related streaming telemetry performance monitoring parameters
+like input power, output power, bias current and so on.
+
+## Setup
+Optics test requirements alongside the platform FNT require 2 optics DUT
```
Comment (ejbrever) [inline feature/platform/tests/optics_power_and_bias_current_test/README.md:9]: what is FNT? Perhaps we can define that here?

File: feature/platform/tests/optics_power_and_bias_current_test/README.md [hunk 2]
```diff
@@ -1,49 +1,102 @@
...
-    validate following leafs are set:
+*   Step 2: Get list of components of type TRANSCEIVER. Verify the instant value is
+    between the corresponding lower and upper thresholds for both
+    [severity]=WARNING and [severity]=CRITICAL:
+    *   Module case temperature
+        *   /components/component/transceiver/thresholds/threshold/state/module-temperature-lower
+        *   /components/component/transceiver/thresholds/threshold/state/module-temperature-upper
+        *   /components/component/transceiver/thresholds/threshold/state/severity
+        *   /components/component/state/temperature/instant
+    *   Tx output power
```
Comment (ejbrever) [inline feature/platform/tests/optics_power_and_bias_current_test/README.md:46]: Maybe we can call out that every physical channel is expected? For example if there are four physical channels in the transceiver we must see all of them.

This also applies to RX below.

Comment (ejbrever) [inline feature/platform/tests/optics_power_and_bias_current_test/README.md:46]: Do we also plan to add total aggregate channel powers in this doc?

For example:
/components/component/optical-channel/state/input-power

File: feature/platform/tests/optics_power_and_bias_current_test/README.md [hunk 3]
```diff
@@ -1,49 +1,102 @@
-# gNMI-1.13: Telemetry: Optics Power and Bias Current
+# gNMI-1.13: Optics Telemetry, Instant, threshold, and miscellaneous static info
```
Comment (ejbrever) [inline feature/platform/tests/optics_power_and_bias_current_test/README.md:1]: Would a different test check for min/max/avg values?

File: feature/platform/tests/optics_power_and_bias_current_test/README.md (hunk 1 above)
Comment (ahsaanyousaf) [reply feature/platform/tests/optics_power_and_bias_current_test/README.md:9]: updated to functional tests

File: feature/platform/tests/optics_power_and_bias_current_test/README.md (hunk 2 above)
Comment (ahsaanyousaf) [reply feature/platform/tests/optics_power_and_bias_current_test/README.md:46]: Updated to call out multiple lanes.

Comment (ahsaanyousaf) [reply feature/platform/tests/optics_power_and_bias_current_test/README.md:46]: These tests are for gray optics not for 400ZR so there is only single input power value no concept of signal power here

File: feature/platform/tests/optics_power_and_bias_current_test/README.md (hunk 3 above)
Comment (ahsaanyousaf) [reply feature/platform/tests/optics_power_and_bias_current_test/README.md:1]: This is for gray optics only. Plan is to request min/max/avg for ZR

**Review by ejbrever — APPROVED** [review]
LGTM

**Review by dplore — APPROVED** [review]

**Pull request merged**